packages/workflow/src/NodeHelpers.ts

```diff
@@ -351,6 +352,58 @@ const declarativeNodeOptionParameters: INodeProperties = {
 	],
 };
 
+/**
+ * Determines if the node is of INodeType
+ */
+export function isINodeType(obj: unknown): obj is INodeType {
+	return typeof obj === 'object' && obj !== null && 'execute' in obj;
+}
+
+/**
+ * Modifies the description of the passed in object, such that it can be used
+ * as an AI Agent Tool.
+ * Returns the modified item (not copied)
+ */
+export function convertNodeToAiTool<
```

Comment:
why did we decide to put this here? can we instead move this to `core`?
We have way much code in the workflow package. So unless some code actually needs to be shared between backend, frontend, and nodes, we should avoid adding it to this package 🙏🏽